### PR TITLE
OKD adding EOL banner

### PR DESCRIPTION
**Pull request by mburke5678**
Request to add the _You are viewing documentation for a release that is no longer maintained._  banner to the unsupported OKD docs.

### CODE REVIEW

File: _templates/_page_openshift.html.erb [hunk 1]
```diff
@@ -130,6 +140,36 @@
           <% else %>
             <%= breadcrumb_root %>
```
Comment (gaurav-nelson) [inline _templates/_page_openshift.html.erb:141]: To fix the issues replace this line with 
```
<% if (distro_key == "openshift-origin") %>
                <a href="https://docs.okd.io/<%= version %>/welcome/index.html">
              <%= distro %>
              </a>
              <select id="version-selector" onchange="versionSelector(this);">
                <option value="latest">latest</option>
                <option value="4.11">4.11</option>
                <option value="4.10">4.10</option>
                <option value="4.9">4.9</option>
                <option value="4.8">4.8</option>
                <option value="4.7">4.7</option>
                <option value="4.6">4.6</option>
                <option value="4.5">4.5</option>
                <option value="4.4">4.4</option>
                <option value="4.3">4.3</option>
                <option value="4.2">4.2</option>
                <option value="4.1">4.1</option>
                <option value="3.11">3.11</option>
                <option value="3.10">3.10</option>
                <option value="3.9">3.9</option>
                <option value="3.7">3.7</option>
                <option value="3.6">3.6</option>
              </select>
              <% else %>
                <%= breadcrumb_root %>
              <% end %>
```

**Comment by aireilly** [comment]
LGTM

File: _templates/_page_openshift.html.erb [hunk 2]
```diff
@@ -128,9 +138,34 @@
...
+                <option value="4.10">4.10</option>
+                <option value="4.9">4.9</option>
+                <option value="4.8">4.8</option>
+                <option value="4.7">4.7</option>
+                <option value="4.6">4.6</option>
+                <option value="4.5">4.5</option>
+                <option value="4.4">4.4</option>
+                <option value="4.3">4.3</option>
+                <option value="4.2">4.2</option>
+                <option value="4.1">4.1</option>
```
Comment (gaurav-nelson) [inline _templates/_page_openshift.html.erb:158]:
```suggestion
```

These are not OKD versions and should be removed

**Comment by gaurav-nelson** [comment]
@mburke5678 Just one last comment. Once we remove those versions, we can get this merged 🍔

**Pull request merged**